### PR TITLE
Generated Guidelines XML from the CSV file

### LINKED ISSUE #123: Convert existing csv into xml
Currently we have a source file in CSV format that needs to be converted to XML.

The current file exists on a branch [here](https://github.com/IntelliTect/CodingGuidelines/blob/EssentialCSharpGuidelinesCSV/EssentialCSharpGuidelines/Guidelines(8th%20Edition).csv).

We should convert it into xml that is similar to the following:
```xml
<?xml version="1.0" encoding="UTF-8" ?>
<root>
    <guideline key="Ch01_fa67753" severity="DO" section="Naming" subsection="Variables and Fields">
DO favor clarity over brevity when naming identifiers.
    </guideline>
</root>
```

Be aware there are some non-standard characters in the csv (like smart quotes) that will need to be converted.
This can be done with a script or by hand, once this is done we will not be using the csv again.

### PR DESCRIPTION
**Pull request by frosch95**
Closes #123

### CODE REVIEW

File: docs/Guidelines(8th Edition).xml [hunk 1]
```diff
@@ -0,0 +1,251 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+	<guideline key="Ch01_fa67753" severity="DO" section="Naming" subsection="Variables and fields">DO favor clarity over brevity when naming identifiers.</guideline>
+	<guideline key="Ch01_674fe1e" severity="DO NOT" section="Naming" subsection="Abbreviations">DO NOT use abbreviations or contractions within identifier names.</guideline>
+	<guideline key="Ch01_024aa0c" severity="UNKNOWN ARGL" section="Naming" subsection="Abbreviations">"DO NOT use any acronyms unless they are widely accepted, and even then use them consistently."</guideline>
```
Comment (Keboo) [inline docs/Guidelines(8th Edition).xml:5]: It looks like the severity on several of these items is set to UNKNOWN ARGL. From looking over the items it appears that it is likely caused by the leading quotation marks

Comment (frosch95) [reply docs/Guidelines(8th Edition).xml:5]: @Keboo oh you are right, with the quotes the automatic recognition of the severtiy didn't work correctly. I will fix this.

Comment (frosch95) [reply docs/Guidelines(8th Edition).xml]: @Keboo done

**Review by Keboo — APPROVED** [review]
Looks great, thank you.

**Comment by frosch95** [comment]
You are welcome :-)

**Pull request merged**